src/main/java/no/idporten/userservice/api/EIDResponse.java

```diff
@@ -0,0 +1,23 @@
+package no.idporten.userservice.api;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.Instant;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class EIDResponse {
+
+    private String name;
+
+    private Instant lastLogin;
```

Comment:
Bruke JsonProperty for å gi snake_case_attribute_names i responser?

Reply:
fixed